src/tests/hash_functions/test_32_bit_hash_performance.f90

```diff
@@ -0,0 +1,190 @@
+program test_32_bit_hash_performance
+!! Program to compare the relative performance of different 32 bit hash
+!! functions
```

Comment:
Are there also tests of correctness? We would want the tests to fail if a bug is introduced.

Reply:
I have tests for correctness that compare with the output of the corresponding C codes, but the tests are outside the standard library as they require invoking C and C++ compilers.